Makefile.am

```diff
@@ -1,12 +1,17 @@
 ACLOCAL_AMFLAGS = -I libltdl/m4
 
-SUBDIRS = src/libmongoc
+SUBDIRS =
+
+if BUILD_WITH_OWN_LIBMONGOC
```

Comment:
Dumb question: where is `BUILD_WITH_OWN_LIBMONGOC` specified? I saw that https://github.com/Stackdriver/agent-packaging/pull/37/files seems to set `--enable-mongodb --with-libmongoc=own`. Then how it got converted to this flag? Am I missing anything?

Reply:
https://github.com/Stackdriver/collectd/blob/stackdriver-agent-5.5.2/configure.ac#L3047

Reply:
Good to know. THX!